torchvision/io/video.py

```diff
@@ -287,15 +302,15 @@ def _can_read_timestamps_from_packets(container):
     return False
 
 
-def _decode_video_timestamps(container):
+def _decode_video_timestamps(container: av.container.Container) -> List[int]:
     if _can_read_timestamps_from_packets(container):
         # fast path
         return [x.pts for x in container.demux(video=0) if x.pts is not None]
     else:
         return [x.pts for x in container.decode(video=0) if x.pts is not None]
 
 
-def read_video_timestamps(filename, pts_unit="pts"):
+def read_video_timestamps(filename: str, pts_unit: str = "pts") -> Tuple[List[int], int]:
```

Comment:
This docstring states that `video_fps` should be `int` where as the docstring of [`_read_video_from_file`](https://github.com/pytorch/vision/blob/df6a7960920534d53d31f6d06fa89dea90cc1b50/torchvision/io/_video_opt.py#L216-L218) states that it should be `float`. Which one is correct?

Reply:
@fmassa This is the last piece before `mypy` is happy.

Reply:
We do
```python
video_fps = float(video_stream.average_rate)
```
so we return a float.